src/java.xml/share/classes/com/sun/org/apache/xerces/internal/impl/msg/XMLMessages_de.properties

```diff
@@ -323,3 +326,7 @@
 # Catalog 09
 # Technical term, do not translate: catalog
         CatalogException=JAXP00090001: CatalogResolver ist mit dem Katalog "{0}" aktiviert, eine CatalogException wird jedoch zurückgegeben.
+
+# Implementation Property DTD
+        JDK_DTD_DENY = JAXP00010008: DOCTYPE ist nicht zulässig, wenn die DTD-Eigenschaft auf "Ablehnen" gesetzt wurde. Weitere Informationen: Eigenschaft jdk.xml.dtd.support in java.xml/module-summary.
```

Comment:
This version quoted "Ablehnen" while the other two (ja - "拒否" and zh_CN - "拒绝") didn't. If we want to be consistent, "deny" would be better since that's the literal value. The English version should have quoted "deny".

The previous translations in these files have not been consistent. Some of the key words were translated. If we want to keep this translation as is, it's probably better to remove the quote in the "de" file.

Reply:
removed quotes around Ablehnen